src/test/java/org/opensearch/security/dlic/rest/api/SecurityConfigApiActionTest.java

```diff
@@ -36,31 +36,47 @@ public SecurityConfigApiActionTest() {
     }
 
     @Test
-    public void testSecurityConfigApiRead() throws Exception {
+    public void testSecurityConfigApiReadForSuperAdmin() throws Exception {
 
         setup();
 
         rh.keystore = "restapi/kirk-keystore.jks";
         rh.sendAdminCertificate = true;
 
-        HttpResponse response = rh.executeGetRequest(ENDPOINT + "/securityconfig", new Header[0]);
+        verifyResponsesWithoutPermissionOrUnsupportedFlag();
+    }
+
+    @Test
+    public void testSecurityConfigApiReadRestApiUser() throws Exception {
+
+        setupWithRestRoles();
+
+        rh.keystore = "restapi/kirk-keystore.jks";
+        rh.sendAdminCertificate = false;
+
+        final var restApiHeader = encodeBasicHeader("test", "test");
+        verifyResponsesWithoutPermissionOrUnsupportedFlag(restApiHeader);
+    }
+
+    private void verifyResponsesWithoutPermissionOrUnsupportedFlag(final Header... headers) {
+        HttpResponse response = rh.executeGetRequest(ENDPOINT + "/securityconfig", headers);
         Assert.assertEquals(HttpStatus.SC_OK, response.getStatusCode());
 
-        response = rh.executePutRequest(ENDPOINT + "/securityconfig", "{\"xxx\": 1}", new Header[0]);
+        response = rh.executePutRequest(ENDPOINT + "/securityconfig", "{\"xxx\": 1}", headers);
         Assert.assertEquals(HttpStatus.SC_METHOD_NOT_ALLOWED, response.getStatusCode());
```

Comment:
should we add a test here to check for FORBIDDEN response for `PUT /securityconfig/config`

Reply:
there are tests for it:
`testSecurityConfigApiReadForSuperAdmin` and `testSecurityConfigApiReadRestApiUser`